setup/login-alerts.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Create a script to be called when a user logs in
+cat << 'EOF' > /etc/ssh/login-alert.sh
+#!/bin/bash
```

Comment:
These two comments are unnecessary.  (or maybe the "Create a script" should be reworded?)